modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestraySuitesAdministrator.testcase

```diff
@@ -187,7 +187,7 @@ definition {
 				requirementName = "ATF_Requirements Summary");
 		}
 
-		task ("And given that the user adds a Smart Suite") {
+		task ("And given that the user add a Smart Suite") {
```

Comment:
The old form "adds" is the correct one in this case. @joelrocha68 would you mind to change it back to "adds"?